### PR TITLE
Add roommates

### PR DESCRIPTION
**Pull request by CookinLima**
managed to add flatmates using ajax. 

Encountered a problem where "+" button automatically disables after adding a flatmate. If you're trying this feature, you need to refresh page (lmao, the irony) for the button to be active again. 

1) Flatmate needs to have an existing user account to be added. 
2) Flatmate automally gets directed to chores page after logging in

### CODE REVIEW

**Review by employmee — APPROVED** [review]
other than the minor mistake in the db seed, lookin good. thanks!!
